CMakeLists.txt

```diff
@@ -223,6 +226,18 @@ if(USE_GAMEMATH)
     endif()
 endif()
 
+if(USE_SDL2)
+    include(FetchContent)
+
+    FetchContent_Declare(
+        SDL2
+        GIT_REPOSITORY https://github.com/libsdl-org/SDL.git
```

Comment:
What happens if `libsdl-org/SDL.git` goes offline? Will Thyme break? If yes, what can we do to not have external dependency?

Reply:
GitHub & SDL2 are both big organization and i doub't they'll just go "offline". We already rely on GitHub repos from AssemblyArmada. When organizations are renamed GitHub will auto redirect. 

As an alternative the repo could be forked, but as i said i don't see a requirement for that